app/uk/gov/hmrc/plasticpackagingtax/registration/models/request/AmendmentJourneyAction.scala

```diff
@@ -102,3 +102,17 @@ class AmendmentJourneyAction @Inject() (
 object AmendmentJourneyAction {
   val SessionId = "sessionId"
 }
+
+@ImplementedBy(classOf[AmendmentJourneyActionImpl])
+trait AmendmentJourneyAction extends ActionRefiner[AuthenticatedRequest, JourneyRequest] {
+
+  def updateLocalRegistration(updateFunction: Registration => Registration)(implicit
+    request: AuthenticatedRequest[Any]
+  ): Future[Registration]
+
+  def updateRegistration(updateFunction: Registration => Registration)(implicit
+    request: AuthenticatedRequest[Any],
+    headerCarrier: HeaderCarrier
+  ): Future[SubscriptionCreateOrUpdateResponse]
+
+}
```

Comment:
I'm not sure why you've done this, related to the use of fakes in controller unit tests?

Reply:
Yeah provides an interface